CMakeLists.txt

```diff
@@ -168,6 +169,10 @@ else()
     set(RUNTIME_DIR_DEFAULT "${CMAKE_INSTALL_FULL_LOCALSTATEDIR}/run/sddm")
 endif()
 
+# utmps
+if (USE_UTMPS)
```

Comment:
IMO this should be set as optional and get used automatically if found.

Reply:
Do I understand correctly that utmps should be used if found even if user didn't request it?

Reply:
If it's available, it's on the user's system and it should probably be used. Do you see any reason not to?

Reply:
I concur. I've just asked to make sure I understand you correctly.